src/main/kotlin/baclkjack/controller/BlackJackController.kt

```diff
@@ -0,0 +1,40 @@
+package baclkjack.controller
+
+import baclkjack.domain.BlackJackGame
+import baclkjack.view.InputView
+import baclkjack.view.ResultView
+
+class BlackJackController(
+    private val inputView: InputView = InputView,
+    private val resultView: ResultView = ResultView
+) {
+
+    fun play() {
+        val players = inputView.inputPlayer()
+        resultView.showHit(players.joinToString { name -> name })
```

Comment:
굳이 name → name을 하지 않아도 될 것 같아요~

```suggestion
        resultView.showHit(players.joinToString())
```